python/smqtk/tests/algorithms/nn_index/test_NNI_MRPT.py

```diff
@@ -0,0 +1,220 @@
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+
+from six.moves import range, zip
+
+import random
+import unittest
+
+import nose.tools as ntools
+import numpy as np
+
+import os.path as osp
+
+from smqtk.representation.descriptor_element.local_elements import \
+    DescriptorMemoryElement
+from smqtk.algorithms import get_nn_index_impls
+from smqtk.algorithms.nn_index.mrpt import MRPTNearestNeighborsIndex
+from smqtk.representation.descriptor_index.memory import MemoryDescriptorIndex
+from smqtk.utils.errors import ReadOnlyError
+
+
+# Don't bother running tests of the class is not usable
+if MRPTNearestNeighborsIndex.is_usable():
```

Comment:
This is not needed since MRPT's ``is_usable`` always returns true.

Reply:
Done.